### PR TITLE
honeybadger error fixes

### PR DESCRIPTION
**Pull request by kkoskelin**
[deleted block about CSP and `unsafe-eval`]

These changes address errors I've seen in the browser; refusal to load `site.manifest`, work queue state variables unset which result in `messages` page being blank, and then one not in the browser -- a missing null-check.

### CODE REVIEW

**Review by rachaelparris — COMMENTED** [review]
We haven't actually seen any errors in the UI related to this. My opinion is that unless we see an error in the UI, an error in the console/HB doesn't necessarily mean we need to fix anything. I think we will have to have some tolerance for errors to keep our CSP locked down as much as possible. I'm open to other opinions, though, and would love to hear input from others!

**Comment by kkoskelin** [comment]
@rachaelparris your comments are noted. I've removed the change and the explanation.  This PR now contains a *different* CSP change, however, which welcomes your input.

File: web-client/src/presenter/actions/chooseWorkQueueAction.js [hunk 1]
```diff
@@ -28,7 +28,8 @@ export const chooseWorkQueueAction = ({ get, path, props, store }) => {
     store.set(state.workQueueToDisplay.box, props.box);
   }
 
-  let queuePrefs = get(state.workQueueToDisplay);
+  const defaultQueue = { box: 'inbox', queue: 'my', workQueueIsInternal: true };
+  const queuePrefs = { ...defaultQueue, ...get(state.workQueueToDisplay) };
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/chooseWorkQueueAction.js]: Could you add test coverage for this?

File: shared/src/persistence/elasticsearch/getIndexNameForRecord.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ exports.getIndexNameForRecord = record => {
   ];
 
   const isRecordOfType = (record, type) => {
-    if (record.entityName && record.entityName.S) {
+    if (record && record.entityName && record.entityName.S) {
```
Comment (rachaelparris) [inline shared/src/persistence/elasticsearch/getIndexNameForRecord.js:12]: Could you add a test for this?

**Review by rachaelparris — APPROVED** [review]
Just needs a couple tests!

**Pull request merged**